easybuild/easyconfigs/i/ImageJ/ImageJ-1.5.1i.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'ImageJ'
+version = '1.5.1i'
+
+homepage = 'https://imagej.nih.gov/ij'
+description = "Image Processing and Analysis in Java"
```

Comment:
Okay, good to be educated before leaving the flat on a Saturday morning...

Looks fine.